pkg/cmd/pipeline/translate.go

```diff
@@ -382,6 +382,13 @@ func AddDevAnnotations(ctx context.Context, manifest *model.Manifest, c kubernet
 			oktetoLog.Infof("could not add %s dev annotations due to: %s", devName, err.Error())
 			continue
 		}
+		if repo != "" && app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation] == repo {
+			continue
+		}
+		if app.ObjectMeta().Annotations[model.OktetoDevNameAnnotation] != "" {
+			continue
+		}
```

Comment:
```suggestion
                 annotationRepo := app.ObjectMeta().Annotations[model.OktetoRepositoryAnnotation]
		if  annotationRepo  == repo && repo != "" {
			continue
		}
```

both have to be the same and not empty ? could we reduce conditions like that?

Reply:
I think you are removing the condition to check `model.OktetoDevNameAnnotation` annotation. Maybe I'm seeing wrong the diff you provide, but each if is checking a different annotation

Reply:
omg, you are right. my mistake